README.md

```diff
@@ -906,11 +907,54 @@ internal class Settings
 Once we define our settings we can access them using the `Global` property inside `Instance`.
 ```csharp
 //Get user from settings
-var user = SettingsProvider<Settings>.Instance.Global.User
+var user = SettingsProvider<Settings>.Instance.Global.User;
 
  //Modify the port 
  SettingsProvider<Settings>.Instance.Global.Port = 20;
  
  //if we want those settings to persist
  SettingsProvider<Settings>.Instance.PersistGlobalSettings();
 ```
+
+### The `Connection` class
+It represents a network connection with 2 operation modes (server or client side). It wraps a TcpClient and its corresponding network streams.
```

Comment:
It represents a wrapper for `TcpClient` (a TCP network connection) either on the server or on the client. It provides access to the input and output network streams. It is capable of working in 2 modes.